docs/Customs Declare Exports AutoComplete.js

```diff
@@ -1,7 +1,7 @@
 // ==UserScript==
 // @name         Customs Declare Exports AutoComplete
 // @namespace    http://tampermonkey.net/
-// @version      1.18
+// @version      1.20
```

Comment:
Why jump to 1.20? Not 1.19

Reply:
Really dunno - it should just have to be higher the previews